src/unix/lwt_engine.mli

```diff
@@ -128,11 +128,24 @@ end
 (** {2 Predefined engines} *)
 
 type ev_loop
+
+module Ev_backend :
+sig
+  type t
+  val default : t
+  val select : t
+  val poll : t
+  val epoll : t
+  val kqueue : t
+  val devpoll : t
+  val port : t
+end
+
```

Comment:
It would be convenient to have a `pp` function here to have a human-readable representation of the abstract type, for logging purposes.